src/services/api-service.js

```diff
@@ -0,0 +1,15 @@
+import stringToObject from './stringToObject';
+
+export const apiCall = (url, method, body) => {
+  const options = {
+    headers: { 'Content-Type': 'application/json' },
+  };
+  
+  if(method === 'post') {
```

Comment:
You need to handle "PATCH" and "PUT" in the same way - these will both be taking a JSON body with which to update.